interface/src/app/home/home.tpl.html

```diff
@@ -1,6 +1,40 @@
 <div>
   <h1>ADAGE: Home</h1>
-  <p class="lead">
-    This shell is the framework into which the ADAGE server will grow.
-  </p>
+  <div class="lead">
+    <h2>ADAGE introduction</h2>
+    <p>
+      ADAGE (or eADAGE) models are neural networks trained in an unsupervise
```

Comment:
@tj8901nm: by the way, it looks like there is a typo in "...unsupervise" with no "d" at the end in the README of ADAGEpath itself. I will correct it here.